packages/tradershub/src/constants/regulators-modal-content.ts

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable sort-keys */
+export type TRowItem = {
+    options?: Record<string, boolean | string>;
+    text: string;
+};
+type TRegulatorContent = Record<string, TRowItem | TRowItem[]>;
+
+export type TRegulatorsContentProps = {
+    attribute: string;
+    content: TRegulatorContent;
+    id: string;
+    idx?: number;
+};
+
+export const getCFDContents: TRegulatorsContentProps[] = [
```

Comment:
No need to type it, the type can be inferred from the value.

Reply:
@mahdiyeh-deriv I can't, because I need this type for Row component:

const Row = ({ attribute, content, id, idx }: TRegulatorsContentProps) => (

and I can't write like this:

const Row = ({ attribute, content, id }: typeof getCFDContents) => (

because TS is angry with it and doesn't know what is attribute, content and id